### ISSUE TITLE
Browser testing issues

### ISSUE THREAD

**Issue by urlsangel** [freedomofpress/pressfreedomtracker.us#1307]
@harrislapiroff 

I'm going to put all of my findings into one issue for now to avoid polluting Github, they can then be pulled out into separate issues based on priority etc.

**Comment by urlsangel** [comment]
### ~Social media nav in footer too close to footer links~

Resolved #1386 

* Desktop/Chrome/Mac
* https://redesign.pressfreedomtracker.us/

<img width="813" alt="Screenshot 2022-03-29 at 09 49 42" src="https://user-images.githubusercontent.com/560463/160573405-c443c39c-65f3-4cca-af49-7f6cb3c3ffa5.png">

* Mobile/Chrome/Galaxy 22
* https://redesign.pressfreedomtracker.us/

<img width="276" alt="Screenshot 2022-03-29 at 14 29 29" src="https://user-images.githubusercontent.com/560463/160622506-21cbd4be-95a5-4e5e-b58e-57d2c45ebc26.png">

**Comment by urlsangel** [comment]
### ~Mobile nav icon margin incorrect~

**Edit: resolved here:** https://github.com/freedomofpress/pressfreedomtracker.us/pull/1399

* Mobile/Chrome/Mac (replicated on mobile device via BS)
* https://redesign.pressfreedomtracker.us/

Figma has the margins as `margin-top: 8px` and `margin-right: 16px`, the actual is `17px` and `33px` respectively.

<img width="285" alt="Screenshot 2022-03-29 at 10 12 03" src="https://user-images.githubusercontent.com/560463/160578114-88d8682b-43f3-47f4-b8ff-b3a0dc2ff8e8.png">

**Comment by urlsangel** [comment]
### Chart filters spacing appears inconsistent

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/

The Figma design doesn't have stacked filters (not sure what was meant in the design), but the rows look a little close together, and are missing an overline (which may lead to it feeling like a big gap above the top filter buttons row).

<img width="379" alt="Screenshot 2022-03-29 at 10 19 38" src="https://user-images.githubusercontent.com/560463/160580780-d48e5262-5459-4be8-a5e8-b05a1eafa6e3.png">

**Comment by urlsangel** [comment]
### ~Button bottom border and search placeholder overlaps~

**Edit: fixed**

* Safari/iPhone 13
* https://redesign.pressfreedomtracker.us/

The bottom border overlap looks like it appears wherever `btn-primary` is used.

<img width="266" alt="Screenshot 2022-03-29 at 10 26 39" src="https://user-images.githubusercontent.com/560463/160581148-21be2c48-8135-434a-a94d-1702f766d118.png">

<img width="159" alt="Screenshot 2022-03-29 at 10 35 38" src="https://user-images.githubusercontent.com/560463/160582167-cd3f68cf-cb80-43a3-9b30-b7def3c8ba0b.png">

**Comment by urlsangel** [comment]
### Overflow X movement on mobile

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/

There appears to be available movement on the x axis when scrolling, although I can't find anything overlapping the container, and the content seems to snap back to position. This makes the page feel very floaty when scrolling through content.

Maybe add `overflow-x: hidden` to the `body`?

**Comment by urlsangel** [comment]
### Double underline on incident card in "Recent incidents" block

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/

Should `last-of-type` on these cards lose the bottom border?

<img width="275" alt="Screenshot 2022-03-29 at 10 58 08" src="https://user-images.githubusercontent.com/560463/160586222-d994a1ec-10af-435d-a15c-be50facf6e44.png">

**Comment by urlsangel** [comment]
### Footer sign up form fields alignment and border

**Edit: partially fixed.** The curve has been fixed using `border-radius: 0`, the fields going beyond the container could be fixed by `box-sizing: border-box`, but this would have potential knock on effects to other areas of the site so needs regression testing if implemented.

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/

The left bottom border has a slight curve, and the fields extend to the edge of the screen - they should have some right margin according to Figma.

<img width="291" alt="Screenshot 2022-03-29 at 10 59 56" src="https://user-images.githubusercontent.com/560463/160586855-42bde555-2383-4f4f-aeb5-d45dcbbeda0e.png">

**Comment by urlsangel** [comment]
### Text wrap on "Submit an incident" button means icon is misaligned

- Safari/iPhone 13 Mini
- https://redesign.pressfreedomtracker.us/

This is the view of the buttons when the mobile nav is expanded - wrapping occurs on iPhone Mini size screens.

<img width="273" alt="Screenshot 2022-03-29 at 11 08 30" src="https://user-images.githubusercontent.com/560463/160589270-211a9eaa-f100-4581-b3de-9be731c11601.png">

**Comment by urlsangel** [comment]
### Mobile nav buttons very far away on mobile menu via tablet

**Harris:** Opened https://github.com/freedomofpress/pressfreedomtracker.us/issues/1342 to address. Let's leave where they are, but add margins on the bottom

***

- Safari/iPad 9th
- https://redesign.pressfreedomtracker.us/

The "Donate" and "Submit an incident" buttons are very far away from the main navigation, maybe they should be part of the nav rather than absolutely positioned to the container?

<img width="395" alt="Screenshot 2022-03-29 at 11 24 47" src="https://user-images.githubusercontent.com/560463/160591704-5bc9b98e-bb16-4b42-a5e2-4f7b81a1b3c7.png">

**Comment by urlsangel** [comment]
### ~Search field has rounded border on iPad Pro~

**Edit: fixed**

- Safari/iPad Pro 12.9
- https://redesign.pressfreedomtracker.us/

<img width="639" alt="Screenshot 2022-03-29 at 11 33 20" src="https://user-images.githubusercontent.com/560463/160592975-d2735bc8-d7c1-4286-94ab-6b1d684f4bdb.png">

**Comment by urlsangel** [comment]
### Featured incidents block missing margin 

**Harris:** I notice GitHub doesn't support this version of Safari, so it might be below our threshold too.

- Safari/iPad Air 4
- https://redesign.pressfreedomtracker.us/

<img width="392" alt="Screenshot 2022-03-29 at 11 44 56" src="https://user-images.githubusercontent.com/560463/160594863-2ff1ba3c-4a3a-46c5-8352-b10fc4b09f20.png">

**Comment by urlsangel** [comment]
### Charts misaligned on iPad Air

- Safari/iPad Air/Landscape
- https://redesign.pressfreedomtracker.us/

<img width="558" alt="Screenshot 2022-03-29 at 11 49 13" src="https://user-images.githubusercontent.com/560463/160595535-d0acff8a-18ab-41c8-a649-ebf6a7da0ad7.png">

**Comment by urlsangel** [comment]
### Top nav margin missing between items

- Safari/iPad Air/Landscape
- https://redesign.pressfreedomtracker.us/

<img width="560" alt="Screenshot 2022-03-29 at 11 51 58" src="https://user-images.githubusercontent.com/560463/160595973-1ac1fe90-a5b4-4a52-9ee4-8d550cdcd606.png">

**Comment by urlsangel** [comment]
### Site title wrapped at desktop width

* Firefox/Windows 11
* https://redesign.pressfreedomtracker.us/

<img width="522" alt="Screenshot 2022-03-29 at 15 08 25" src="https://user-images.githubusercontent.com/560463/160630500-e1357193-da9c-4ea3-8cfe-a49685aaa796.png">

**Comment by urlsangel** [comment]
### Download dataset not dismissable by clicking elsewhere

- Chrome/Mac
- https://redesign.pressfreedomtracker.us/all-incidents/

Once revealed by clicking the down arrow control, the details can only be dismissed by clicking again on the arrow icon - this could be improved by closing the details on any document click outside the element. 

<img width="355" alt="Screenshot 2022-03-29 at 15 24 49" src="https://user-images.githubusercontent.com/560463/160634060-b999b674-8131-47e4-8b19-39e87ff3dd92.png">

**Comment by urlsangel** [comment]
### Show less 'X' looks slightly large and a little close to the text

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/all-incidents/ 

<img width="119" alt="Screenshot 2022-03-29 at 16 11 56" src="https://user-images.githubusercontent.com/560463/160644791-1898b17e-e3a5-4b19-90a6-3c844bd36a3a.png">

**Comment by urlsangel** [comment]
### Filter form fields text size small, device zooms in on focus

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/all-incidents/

When attempting to enter text into a filter form, the device zooms into the field which can then make the filters section disorientating - form field font size usually needs to be minimum 16px to avoid this on iOS.

EDIT: this also happens on the search fields at the top of the page.

<img width="278" alt="Screenshot 2022-03-29 at 16 16 13" src="https://user-images.githubusercontent.com/560463/160645624-62ad0474-bf9f-4b16-b8aa-a5c79d56610c.png">

**Comment by urlsangel** [comment]
### "Show xxx incidents" button doesn't do anything if filters already active

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/all-incidents/

In the scenario screenshotted, nothing happens when tapping the "Show 182 incidents" button.

<img width="271" alt="Screenshot 2022-03-29 at 16 20 44" src="https://user-images.githubusercontent.com/560463/160646489-7df58223-c7b3-4bb7-90e0-6764680f089d.png">

**Comment by urlsangel** [comment]
### Filters button ends up over the footer

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/all-incidents/

This would probably make more sense if it stopped at the bottom of the listing container?

<img width="287" alt="Screenshot 2022-03-29 at 16 24 06" src="https://user-images.githubusercontent.com/560463/160647344-5192d45e-07e0-4636-a3ab-eaabfd2faa22.png">

**Comment by urlsangel** [comment]
### Layout of incident card in database search results looks disconnected

- Safari/iPad 9th
- https://redesign.pressfreedomtracker.us/all-incidents/

When an incident has tags, the layout makes it seem quite disconnected (large margin below the title link?). Also, the "Show more" link looks misaligned, as it's indented but doesn't appear aligned with any other element.

<img width="367" alt="Screenshot 2022-03-30 at 10 11 41" src="https://user-images.githubusercontent.com/560463/160796453-260ca1be-425b-4818-9e56-ee011c859d11.png">

**Comment by urlsangel** [comment]
### Image caption on single incident over indented

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/all-incidents/katu-camera-operator-assaulted-camera-damaged-amid-portland-protests/

According to Figma, the caption text should align with the body copy.

<img width="269" alt="Screenshot 2022-03-30 at 11 05 38" src="https://user-images.githubusercontent.com/560463/160807042-3c2713b4-2b2d-443e-955d-03a8aa746371.png">

**Comment by urlsangel** [comment]
### Featured article list missing margin

- Chrome/Mac
- https://redesign.pressfreedomtracker.us/blog/

The article list goes to edge of screen, the Figma design shows it should be indented to follow the headline, other page content etc.

<img width="698" alt="Screenshot 2022-03-30 at 11 48 51" src="https://user-images.githubusercontent.com/560463/160814905-03513953-2ad7-4d79-80ab-bff5b3266f64.png">

**Comment by urlsangel** [comment]
### Featured article items missing right margin

- Safari/iPhone SE 2020
- https://redesign.pressfreedomtracker.us/blog/

<img width="257" alt="Screenshot 2022-03-30 at 11 58 06" src="https://user-images.githubusercontent.com/560463/160816471-cedf45c5-856c-48cd-9e70-fb23252bee99.png">

**Comment by urlsangel** [comment]
### ~Top navigation missing~

**Edit: fixed**

- Safari/iPad Pro 12.9
- https://redesign.pressfreedomtracker.us/blog/

<img width="428" alt="Screenshot 2022-03-30 at 14 56 37" src="https://user-images.githubusercontent.com/560463/160852000-618da067-94af-4492-90cb-69e62517c361.png">

**Comment by urlsangel** [comment]
### Contact page has a "None" button

- Chrome/Mac
- https://redesign.pressfreedomtracker.us/contact/

<img width="711" alt="Screenshot 2022-03-30 at 16 03 13" src="https://user-images.githubusercontent.com/560463/160867180-93a7c750-c8b3-43d5-94c7-f3d5c449fcfa.png">

**Comment by urlsangel** [comment]
### Hover over third column on About page table results in glitchiness

- Chrome/Mac
- https://redesign.pressfreedomtracker.us/about/ 

When hovering over the right side of "See Articles" or "Visit the Page" links, the animation gets confused and jumps between initial and hover state.

<img width="147" alt="Screenshot 2022-03-30 at 16 37 38" src="https://user-images.githubusercontent.com/560463/160874913-bb8cba36-3bb6-49da-bf2b-091cb7b293a2.png">

**Comment by urlsangel** [comment]
### Author page link results in 500 error

- Chrome/Mac
- https://redesign.pressfreedomtracker.us/authors/kirstin-mccudden/

The page linked to from the About template returns a 500 - missing template?

<img width="294" alt="Screenshot 2022-03-30 at 16 40 38" src="https://user-images.githubusercontent.com/560463/160875265-6eaa1647-778b-4b3a-8c54-c682f3ffb904.png">

**Comment by urlsangel** [comment]
### Clicking "Visit the Page" triggers layout jump but no link action

- Safari/iPhone 13
- https://redesign.pressfreedomtracker.us/about/

<img width="272" alt="Screenshot 2022-03-30 at 16 46 02" src="https://user-images.githubusercontent.com/560463/160876609-994fd787-5415-4e57-bf33-d52e24f9ff7d.png">

**Comment by urlsangel** [comment]
### Mobile nav layout broken 

- Safari/iPad Air 4/iPhone SE 2020
- https://redesign.pressfreedomtracker.us

It looks like the mobile nav layout is broken on anything lower than iOS 15 - the above browsers are on 14 and 13 respectively.

<img width="390" alt="Screenshot 2022-04-07 at 11 48 25" src="https://user-images.githubusercontent.com/560463/162183095-732871f2-9a95-4d0b-9899-89fd3993acb6.png">


<img width="262" alt="Screenshot 2022-04-07 at 11 47 23" src="https://user-images.githubusercontent.com/560463/162183061-9cdfdbe6-87a3-4586-9ef7-9d1f19ddf36e.png">
